server/package.json

```diff
@@ -31,6 +31,13 @@
   },
   "devDependencies": {
     "@nestjs/cli": "^9.1.3",
+    "@nestjs/jwt": "^9.0.0",
+    "@nestjs/passport": "^9.0.0",
+    "@types/passport-jwt": "^3.0.7",
+    "@types/passport-local": "^1.0.34",
+    "passport": "^0.6.0",
+    "passport-jwt": "^4.0.0",
+    "passport-local": "^1.0.0",
```

Comment:
We don't use it. At least for now. I'll add it when I need it later.
So could you remove it?